clarifai/client/input.py

```diff
@@ -761,6 +811,65 @@ def upload_annotations(self, batch_annot: List[resources_pb2.Annotation], show_l
 
     return retry_upload
 
+  def patch_annotations(self, batch_annot: List[resources_pb2.Annotation],
+                        action: str = 'merge') -> None:
+    """Upload image annotations to app.
+
+    Args:
+        batch_annot: annot batch protos
+        action (str): Action to perform on the input. Options: 'merge', 'overwrite', 'remove'.
+
+    """
+    if not isinstance(batch_annot, list):
+      raise UserError("batch_annot must be a list of Annotation objects")
+    request = service_pb2.PatchAnnotationsRequest(
+        user_app_id=self.user_app_id, annotations=batch_annot, action=action)
+    response = self._grpc_request(self.STUB.PatchAnnotations, request)
+    response_dict = MessageToDict(response)
+    if response.status.code != status_code_pb2.SUCCESS:
+      try:
+        for annot in response_dict["annotations"]:
+          if annot['status']['code'] != status_code_pb2.ANNOTATION_SUCCESS:
+            self.logger.warning(f"Patch annotations failed, status: {annot['status']}")
+      except Exception:
+        self.logger.warning(f"Patch annotations failed due to {response.status}")
+    else:
+      self.logger.info("\nPatch Annotations Uploaded Successful\n%s", response.status)
+
+  def patch_concepts(self,
+                     concept_ids: List[str],
+                     labels: List[str],
```

Comment:
`labels` param can be None, If not provided we can reuse the `concept_ids` as `labels`

Reply:
Done